content/consul/v1.22.x/content/docs/reference/cli/consul-k8s.mdx

```diff
@@ -277,6 +278,207 @@ global:
   name: consul
 ```
 
+### `debug`
+
+The debug command collect configs, logs, and other data from a Consul on Kubernetes deployment in a archive/dir.
+
+```shell-session
+$ consul-k8s debug <OPTIONS>
+```
+
+The following Command options are available.
+
+| Flag                            | Description                                                                                                                                                                                                                                                                                                                                                                                       | Default                                                     |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------|
+| <nobr>`-duration`, `-d`</nobr>  | `time.Duration` Sets the time duration to capture logs of consul cluster. (e.g., '5m' for the logs of next 5 minutes). <br/> Minimum time duration should be atleast 10s.                                                                                                                                                                                                                         | `5m0s`                                                      |
+| <nobr>`-since`, `-s`</nobr>     | `time.Duration` Sets the time duration from which to start capturing logs from consul cluster. (e.g., '5m' for the logs of last 5 minutes). <br/> Minimum time duration should be atleast 10s.                                                                                                                                                                                                    | none                                                        |
```

Comment:
```suggestion
| <nobr>`-since`, `-s`</nobr>     | `time.Duration` Sets the time duration from which to start capturing logs from the Consul cluster. For example, '5m' for the logs of last 5 minutes. <br/> Minimum time duration should be at least 10s.                                                                                                                                                                                                    | none                                                        |
```